modules/service/src/main/resources/graphql/itc.graphql

```diff
@@ -10363,22 +10369,19 @@ input InstrumentModesInput {
 # Configuration alternatives query
 input SpectroscopyIntegrationTimeInput {
   # Observing wavelength.
```

Comment:
Is the comment correct?

Reply:
Probably not, it was already like that, but it's a good opportunity to improve it. I'll commit a fix.